apps/desktop/src/components/settings/ai/llm/configure.tsx

```
@@ -164,8 +54,11 @@ function HyprProviderCard({
   return (
     <AccordionItem
       value={providerId}
-      className="rounded-xl border-2 border-dashed bg-neutral-50"
       disabled={locked}
+      className={cn([
+        "rounded-xl border-2 bg-neutral-50",
+        true ? "border-solid border-neutral-300" : "border-dashed",
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)